pkgdb2/api/packages.py

```diff
@@ -1287,9 +1291,11 @@ def api_package_request():
             cnt += 1
         collections.append(collection)
 
+    namespaces = pkgdblib.get_status(SESSION, 'namespaces')['namespaces']
```

Comment:
Seems like this should be singular, i.e. ``namespace`` not plural, i.e. ``namespaces``.  Here, and everywhere, no?

There's never a time that a package could have multiple namespaces, correct?